storage/innobase/buf/buf0dblwr.cc

```diff
@@ -385,8 +386,11 @@ void buf_dblwr_t::recover()
     fil_space_t *space= fil_space_t::get(space_id);
 
     if (!space)
+    {
       /* The tablespace that this page once belonged to does not exist */
+      encrypted_pages.push_back(*i);
       continue;
+    }
```

Comment:
It would be good to add a comment that the `FIL_PAGE_LSN` of these pages will be validated in `recover_encrypted_page()`.